### PR TITLE
naufil-css-positioning

### PR DESCRIPTION
**Pull request by Naufil-Shemle**

### CODE REVIEW

File: tasks-by-users/Naufil-Shemle/03-css-positioning/index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            accusantium laboriosam. Commodi hic facilis et nesciunt, totam
+            mollitia ullam quam dolorem ratione explicabo porro. Dolorum nostrum
+            quidem, adipisci modi fugit molestiae voluptatem voluptas excepturi
+            ab in molestias hic debitis cum eos cupiditate vero sequi suscipit
+            expedita. Neque iste ipsa optio nobis dolorem earum laboriosam omnis
+            labore voluptas ab voluptatum, impedit qui ipsum vel eligendi
+            delectus eum quo, repudiandae minima distinctio. Illo nesciunt quae
+            veritatis, ducimus blanditiis velit sunt!
+          </p>
+          <img src="cloud image.jpg" alt="Cloud image" width="100%" />
```
Comment (Abbas4tech) [inline tasks-by-users/Naufil-Shemle/03-css-positioning/index.html:114]: Remove inline styling @Naufil-Shemle

File: tasks-by-users/Naufil-Shemle/03-css-positioning/index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        <li>link</li>
+        <li>link</li>
+        <li>link</li>
+        <li>link</li>
+      </ul>
+    </nav>
+
+    <div class="wrapper">
+      <div id="banner">
+        <img src="cloud image.jpg" alt="cloud image" width="100%" />
```
Comment (Abbas4tech) [inline tasks-by-users/Naufil-Shemle/03-css-positioning/index.html:22]: Remove inline styling @Naufil-Shemle

File: tasks-by-users/Naufil-Shemle/03-css-positioning/style.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
+body {
+  background: #c2c2c2;
+}
+.wrapper {
+  background: #fff;
+  width: 960px;
+  margin: 0 auto;
+  padding: 20px;
```
Comment (Abbas4tech) [inline tasks-by-users/Naufil-Shemle/03-css-positioning/style.css:8]: Group Similar properties @Naufil-Shemle

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: tasks-by-users/Naufil-Shemle/03-css-positioning/index.html (hunk 1 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/index.html:114]: Done

File: tasks-by-users/Naufil-Shemle/03-css-positioning/index.html (hunk 2 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/index.html:22]: Done

File: tasks-by-users/Naufil-Shemle/03-css-positioning/index.html (hunk 1 above)
Comment (Abbas4tech) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/index.html:114]: Inline styling still there, there is no recent commit for this. why you resolved?? @Naufil-Shemle

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: tasks-by-users/Naufil-Shemle/03-css-positioning/index.html (hunk 1 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/index.html:114]: just going commit brother
wait 1 minute

File: tasks-by-users/Naufil-Shemle/03-css-positioning/style.css (hunk 3 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/style.css]: Done

File: tasks-by-users/Naufil-Shemle/03-css-positioning/index.html (hunk 1 above)
Comment (Abbas4tech) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/index.html]: Always Resolve conversation after you made changes suggested by a reviewer. @Naufil-Shemle

**Review by Abbas4tech — APPROVED** [review]

File: tasks-by-users/Naufil-Shemle/03-css-positioning/index.html (hunk 1 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/index.html]: OK Sir

File: tasks-by-users/Naufil-Shemle/03-css-positioning/style.css [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+  width: 100%;
+  background: #333;
+  z-index: 1;
+}
+
+nav ul:after {
+  content: "";
+  display: block;
+  clear: both;
```
Comment (shaikh-7abish) [inline tasks-by-users/Naufil-Shemle/03-css-positioning/style.css:136]: Using these properties 3 times above, use it in once using multiple selector `,`

Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/style.css:136]: I will keep this point from next time.

Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/style.css:136]: At this initial stage I kept this separately

File: tasks-by-users/Naufil-Shemle/03-css-positioning/style.css [hunk 5]
```diff
@@ -0,0 +1,148 @@
+body {
```
Comment (mmusk) [inline tasks-by-users/Naufil-Shemle/03-css-positioning/style.css:1]: It's not a good practise to give properties to body, instead create another class and wrap these properties in it.

**Review by mmusk — APPROVED** [review]

File: tasks-by-users/Naufil-Shemle/03-css-positioning/style.css (hunk 5 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/style.css:1]: As you can you see brother my stuff is in a class but that class size is less than window size that's why I used this.
And I wanted to show the window color and my class color differently

Comment (mmusk) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/style.css:1]: its okay for now but take account of it in future

Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/03-css-positioning/style.css:1]: Inshaallah

**Review by nehalskdev — APPROVED** [review]

**Pull request merged**